fendermint/app/src/cmd/genesis.rs

```diff
@@ -329,6 +332,15 @@ async fn new_genesis_from_parent(
 
     let genesis_info = parent_provider.get_genesis_info(&args.subnet_id).await?;
 
+    let power_scale = if matches!(
+        genesis_info.permission_mode,
+        ipc_api::subnet::PermissionMode::Collateral
+    ) {
+        args.power_scale.unwrap_or(DEFAULT_POWER_SCALE)
+    } else {
+        TokenAmount::DECIMALS as PowerScale
```

Comment:
@cryptoAtwill shouldn't this be 1? We want X federated power to translate directly into X weight, right? Having 18 here would force users to append 18 zeroes to whatever power they actually want to see in the subnet?

Reply:
it's a subtraction here: https://github.com/consensus-shipyard/ipc/blob/d881a2ef505178dbab1cef0409b2515df6419f02/fendermint/vm/genesis/src/lib.rs#L129